recipes-ni/ni-configpersistentlogs/files/ni-configpersistentlogs

```diff
@@ -32,6 +32,11 @@ case $ENABLE_PERSISTENT_LOGS_INI_VAL in
 esac
 
 # This grep command looks for the exact new line.
+if [ ! -f "$FILE_PATH_VOLATILES_CONFIG" ]; then
```

Comment:
Why is this section needed? I see that for sysvinit we install this file in initscripts_1.0.bb - don't we have a systemd replacement that does this?